eos_data_distribution/ndn/test.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- Mode:python; coding: utf-8; c-file-style:"gnu"; indent-tabs-mode:nil -*- */
+#
+# Copyright (C) 2016 Endless Computers INC.
```

Comment:
I think this should be ‘Endless Mobile, Inc.’? At least, nobody’s corrected me on this yet.